content/docs/conditional-rendering.md

```diff
@@ -1,243 +1,245 @@
+En el siguiente ejemplo, crearemos un [componente con estado](/docs/state-and-lifecycle.html#adding-local-state-to-a-class) llamado `GestionInicioSesion`.
 
-It will render either `<LoginButton />` or `<LogoutButton />` depending on its current state. It will also render a `<Greeting />` from the previous example:
+El componente va a renderizar `<BotonInicioSesion />` o `<BotonCierreSesion />` dependiendo de su estado actual. También va a renderizar un `<Saludo />` del ejemplo anterior:
 
 ```javascript{20-25,29,30}
-class LoginControl extends React.Component {
+class GestionInicioSesion extends React.Component {
   constructor(props) {
     super(props);
-    this.handleLoginClick = this.handleLoginClick.bind(this);
-    this.handleLogoutClick = this.handleLogoutClick.bind(this);
-    this.state = {isLoggedIn: false};
+    this.manejarClickInicioSesion = this.manejarClickInicioSesion.bind(this);
+    this.manejarClickCierreSesion = this.manejarClickCierreSesion.bind(this);
+    this.state = {estaConectado: false};
   }
 
-  handleLoginClick() {
-    this.setState({isLoggedIn: true});
+  manejarClickInicioSesion() {
+    this.setState({estaConectado: true});
   }
 
-  handleLogoutClick() {
-    this.setState({isLoggedIn: false});
+  manejarClickCierreSesion() {
+    this.setState({estaConectado: false});
   }
 
   render() {
-    const isLoggedIn = this.state.isLoggedIn;
-    let button;
+    const estaConectado = this.state.estaConectado;
+    let boton;
 
-    if (isLoggedIn) {
-      button = <LogoutButton onClick={this.handleLogoutClick} />;
+    if (estaConectado) {
+      boton = <BotonCierreSesion onClick={this.manejarClickCierreSesion} />;
     } else {
-      button = <LoginButton onClick={this.handleLoginClick} />;
+      boton = <BotonInicioSesion onClick={this.manejarClickInicioSesion} />;
     }
 
     return (
       <div>
-        <Greeting isLoggedIn={isLoggedIn} />
-        {button}
+        <Saludo estaConectado={estaConectado} />
+        {boton}
       </div>
     );
   }
 }
 
 ReactDOM.render(
-  <LoginControl />,
-  document.getElementById('root')
+  <GestionInicioSesion />,
+  document.getElementById('raiz')
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/QKzAgB?editors=0010)
+[**Pruébalo en CodePen**](https://codepen.io/gaearon/pen/QKzAgB?editors=0010)
 
-While declaring a variable and using an `if` statement is a fine way to conditionally render a component, sometimes you might want to use a shorter syntax. There are a few ways to inline conditions in JSX, explained below.
+Si bien declarar una variable y usar una sentencia `if` es una buena forma de renderizar condicionalmente un componente, a veces podrías querer usar una sintaxis más corta. Hay algunas formas de hacer condiciones $INLINE en JSX, explicadas a continuación.
 
-### Inline If with Logical && Operator
+### If $INLINE con operador lógico &&
```

Comment:
What does this `inline` stand out for?

Reply:
If we are not translating `$INLINE`, I think we could say `en la misma línea` or something similar.

Reply:
That title refers to `{isVisible && <Component />}`, instead of using traditional block based ifs or ternary operators. Just FYI.

Reply:
Gotcha. Will take a look at that and translated snippet @123